lib/spack/spack/build_systems/cuda.py

```diff
@@ -13,39 +13,65 @@ class CudaPackage(PackageBase):
     """Auxiliary class which contains CUDA variant, dependencies and conflicts
     and is meant to unify and facilitate its usage.
     """
+    maintainers = ['ax3l', 'svenevs']
```

Comment:
ping @davydden and @alalazo 
wonna help this kitten: :cat2: ?

Reply:
No response, so I'm going to merge this for now. Feel free to add additional maintainers in follow-up commits.